packages/@react-spectrum/calendar/src/CalendarBase.tsx

```diff
@@ -136,12 +136,12 @@ export function CalendarBase<T extends CalendarState | RangeCalendarState>(props
           onClick={() => state.focusNextPage()}
           tabIndex={-1} />
       </VisuallyHidden>
-      {state.validationState === 'invalid' &&
+      {state.isInvalid &&
```

Comment:
```suggestion
      {state.isValueInvalid &&
```
Noticed some breaks in chromatic: https://www.chromatic.com/test?appId=5f0dd5ad2b5fc10022a2e320&id=64e92f45a003125f7a68145c. Running it again locally with the above change, will push if it runs clean